src/views/preview/preview.scss

```diff
@@ -0,0 +1,288 @@
+@import "../../colors";
+@import "../../frameless";
+
+html,
+body,
+#app,
+.gui {
+    margin: 0;
+    width: 100%;
+    height: 100%;
+}
+
+/* override view padding for share banner */
+#view {
+    padding: 0 0 20px 0;
+}
+
+.gui * { box-sizing: border-box; }
+
+.preview {
+    
+    h1 {
```

Comment:
Is there a class we can set on this? Seems a little loose. Also curious why this page has different styles for `h1` from the rest of www?